migrations/20190313122855_betterCoursesTablev3.js

```diff
@@ -0,0 +1,22 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.hasTable('courses').then(function(exists) {
+    if (!exists) {
+      return knex.schema.createTable('courses', function(table) {
```

Comment:
If you want to alter the table, you should use `knex.schema.table`. This won't work since the table aleady exists.